gammapy/scripts/catalog_query.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ..utils.scripts import get_parser
+
+__all__ = ['catalog_query']
+
+def main(args=None):
+    parser = get_parser(catalog_query)
+    parser.add_argument('catalog',
+                        choices=['3FGL'],
+                        help='Catalog for the source e.g. 3FGL')
+    parser.add_argument('source',
+                        help='Source name e.g. J0349.9-2102')
+    parser.add_argument('querytype',
+                        choices=['info', 'lightcurve', 'spectrum'],
+                        help='The query type: info, lightcurve, or spectrum')
```

Comment:
You have to add the following line here:
```python
args = parser.parse_args(args)
```
This will fix this error (that's what it looks like on Python 3 for me):
```
$ gammapy-catalog-query 
Traceback (most recent call last):
  File "/Users/deil/Library/Python/3.4/bin/gammapy-catalog-query", line 9, in <module>
    load_entry_point('gammapy==0.3.dev1392', 'console_scripts', 'gammapy-catalog-query')()
  File "/Users/deil/code/gammapy/gammapy/scripts/catalog_query.py", line 17, in main
    catalog_query(**vars(args))
TypeError: vars() argument must have __dict__ attribute
```